### PR TITLE
Homework8: Ekaterina Tsiunel

### PR DESCRIPTION
**Pull request by ETsiunel**

### CODE REVIEW

**Comment by ETsiunel** [comment]
@Stml89 PR is ready for review.
it was hard but interesting :)

File: hw_8.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+                if delete_count > 1:
+                    return False
+            else:
+                if i == 0 or sequence[i] <= list_sequence[i - 1]:
+                    return False
+    return True
+
+
+sequence = [1, 3, 2, 1]
+print(sequence_check(sequence))
```
Comment (Stml89) [inline hw_8.py]: Using `assert` here is better than `print()`

File: hw_8.py [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        a = int(n/2 + first_number - n)
+    else:
+        a = int(n/2 + first_number)
+    return a
+
+
+n = int(input("Enter n: "))
+while n % 2 == 1:
+    print("n must be even number")
+    n = int(input("Enter n: "))
```
Comment (Stml89) [inline hw_8.py:80]: 2 identical code in line#75 and #78

**Review by Stml89 — CHANGES_REQUESTED** [review]

File: hw_8.py (hunk 1 above)
Comment (ETsiunel) [reply hw_8.py]: fixed

File: hw_8.py (hunk 2 above)
Comment (ETsiunel) [reply hw_8.py:80]: fixed

**Review by Stml89 — APPROVED** [review]

**Pull request merged**